lib/command/src/Obelisk/Command/Deploy.hs

```diff
@@ -94,8 +94,14 @@ deployInit deployOpts root = do
     failWith [i|Deploy directory ${deployDir} should not be the same as project root.|]
   thunkPtr <- wrapNixThunkError (readThunk root) >>= \case
     Right (ThunkData_Packed _ ptr) -> return ptr
-    _ -> wrapNixThunkError (getThunkPtr CheckClean_NotIgnored root Nothing)
+    _ -> wrapNixThunkError $ compat $ getThunkPtr CheckClean_NotIgnored root Nothing
   deployInit' thunkPtr deployOpts
+  where
+#if MIN_VERSION_nix_thunk(0,7,1)
+    compat = fmap $ \(ptr, _isWorktree) -> ptr
+#else
+    compat = id
+#endif
```

Comment:
I guess we should treat a worktree dir the same as a standalone unpacked dir?
@ryantrinkle